src/topology/uniform_space/completion.lean

```diff
@@ -175,7 +175,7 @@ have h_ex : ∀ s ∈ 𝓤 (Cauchy α), ∃y:α, (f, pure_cauchy y) ∈ s, from
 begin
   simp only [closure_eq_cluster_pts, cluster_pt, nhds_eq_uniformity, lift'_inf_principal_eq,
     set.inter_comm _ (range pure_cauchy), mem_set_of_eq],
-  exact (lift'_ne_bot_iff $ monotone_inter monotone_const monotone_preimage).mpr
+  exact (lift'_ne_bot_iff $ (by exact monotone_inter monotone_const monotone_preimage)).mpr
```

Comment:
I'm surprised this change was needed - did you even touch the lemmas used here?

Reply:
Ah, this is still a left-over of the undone change to `monotone`: after I changed the `preorder` to `has_le` I had to change here (and in a couple of other files) proofs to `by exact`.  I am going to undo this change as well, since I have undone the one to `monotone`!

Reply:
I hadn't even noticed that you changed `monotone` in the first place...

Reply:
Oh, that is because I changed it, fixed some things, saw that it broke one of the tests for `apply_fun` and reverted the change!

Reply:
In case you are curious, the change started in `085b1720d9137850ccca940e449f907cb7aa3227`, but was undone a couple of commits after.  I simply found it strange that `monotone` assumes `preorder`, while `strict_mono` assumes `has_lt`.